src/routes/VISPDAT.js

```diff
@@ -1,61 +1,53 @@
 import React, { Component, PropTypes } from 'react';
-import {
-  View,
-  Text,
-  StyleSheet
-} from 'react-native';
 import { connect } from 'react-redux';
 import { bindActionCreators } from 'redux';
+// Actions
 import * as FormActions from '../actions/Form/index.js';
-import SectionHeader from '../components/VISPDAT/SectionHeader.js';
-import TextField from '../components/Login/TextFields.js';
-import RadioButton from '../components/RadioButton.js';
-import Checkbox from '../components/Checkbox.js';
-import DropDown from '../components/DropDown.js';
-
-const styles = Object.assign({}, StyleSheet.create({
-
-  row: {
-    flexDirection: 'row'
-  },
-
-  col1: {
-    flex: 1,
-    flexDirection: 'row',
-    alignItems: 'center',
-    marginLeft: 20, marginRight: 20
-  },
-
-  col2: {
-    flex: 2,
-    flexDirection: 'column',
-    marginLeft: 20, marginRight: 20
-  },
-
-  titleText: {
-    fontSize: 15,
-    fontWeight: 'bold'
-  }
+import FormQuestion from '../components/FormQuestion.js';
+import {
+  ScrollView,
+  Text
+} from 'react-native';
+import { vispdat } from '../utilities/questions.js';
```

Comment:
Probably could alias vispdat to vispdatQuestions to make the code clearer.  Example: `import { foo: bar } from 'cux';` will let you use `bar` instead of foo.